history/characters/10000_orc.txt

```diff
@@ -336,6 +336,7 @@
 		trait = being_undead
 		remove_trait = creature_orc
```

Comment:
Vashj don't change race trait.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/149533719-a14ad90e-9c38-4572-a95d-c9f4f46da121.png)

</details>

Reply:
Also fixed
![image](https://user-images.githubusercontent.com/35965798/149587627-c25052f9-c26f-4fe2-a5b7-6f25b80396a0.png)